addons/mail/static/src/components/chat_window/chat_window.scss

```diff
@@ -54,6 +54,7 @@
 
 .o_ChatWindow_thread {
     flex: 1 1 auto;
+    overflow: hidden;
```

Comment:
This rule is overridden by `auto` from `o_ThreadViewer`, I guess it can be removed.